### PR TITLE
:arrow_up: @okta/samples generator 1.16.0

### PR DESCRIPTION
**Pull request by jmelberg-okta**
Adds default authorization server support.

### CODE REVIEW

File: dependency-reduced-pom.xml [hunk 1]
```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (wdawson-okta) [inline dependency-reduced-pom.xml]: this file should likely be ignored

Comment (wdawson) [inline dependency-reduced-pom.xml]: This file should likely be ignored

File: .gitignore [hunk 2]
```diff
@@ -1,13 +1,9 @@
...
 travis_phantomjs
 .DS_Store
 .vscode
-
-node
-src/main/resources/assets/*
-.idea/
-*.iml
-target/
-dependency-reduced-pom.xml
```
Comment (wdawson) [inline .gitignore]: seems like you want to add these back in

Comment (jmelberg-okta) [inline .gitignore]: Ah. Good catch. I'll update that now.

**Pull request merged**